hc/settings.py

```diff
@@ -81,12 +83,18 @@
 
 # Default database engine is SQLite. So one can just check out code,
 # install requirements.txt and do manage.py runserver and it works
-DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME':   './hc.sqlite',
+if os.environ.get("PLATFORM") == "HEROKU":
+    databse_url = os.environ.get("DATABASE_URL")
+    DATABASES = {
+        'default': dj_database_url.config(default=databse_url)
```

Comment:
Delete [line 87](https://github.com/andela/healthchecks-thunknalla/blob/e88ed78f4a320759e980b2d2408d6ed202d30920/hc/settings.py#L87) and update this to:
```python
'default' = dj_database_url.config(conn_max_age=600)
```